src/ledger/src/multisig.rs

```diff
@@ -0,0 +1,166 @@
+use crate::TargetCommandOpt;
+use clap::Parser;
+use many::message::ResponseMessage;
+use many::server::module::account;
+use many::types::ledger;
+use many::{Identity, ManyError};
+use many_client::ManyClient;
+use minicbor::bytes::ByteVec;
+use tracing::info;
+
+#[derive(Parser)]
+pub struct CommandOpt {
+    #[clap(subcommand)]
+    /// Multisig subcommand to execute.
+    subcommand: SubcommandOpt,
+}
+
+#[derive(Parser)]
+enum SubcommandOpt {
+    /// Submit a new transaction to be approved.
+    Submit {
+        /// The account to use as the source of the multisig command.
+        account: Identity,
+
+        #[clap(subcommand)]
+        subcommand: SubmitOpt,
+    },
+
+    /// Approve a transaction.
+    Approve(TransactionOpt),
+
+    /// Revoke approval of a transaction.
+    Revoke(TransactionOpt),
+
+    /// Execute a transaction.
+    Execute(TransactionOpt),
+}
+
+#[derive(Parser)]
+enum SubmitOpt {
```

Comment:
According to the spec, this could be any `multisig-transaction`.

Reply:
The CLI will not support all the possible types because it becomes a bit insane (and recursive). If you have custom needs, you should use `many` instead.